src/test/java/uk/gov/companieshouse/pscfiling/api/controller/impl/PscIndividualFilingControllerImplIT.java

```diff
@@ -127,7 +128,8 @@ void expectNotFoundResponseWhenPscTypeInvalid() throws Exception {
                         TRANS_ID).content(body).contentType("application/json").headers(httpHeaders))
                 .andDo(print())
                 .andExpect(status().isNotFound())
-                .andExpect(jsonPath("$").doesNotExist());
+                .andExpect(jsonPath("$.errors", hasSize(1)))
+                .andExpect(jsonPath("$.errors[0].error", is(SERVICE_UNAVAILABLE_ERROR)));
```

Comment:
Should these be returning service unavailable errors. Seems like we've changed the way a not found error is handled

Reply:
I'm a bit unsure on how this has suddenly changed but, looking at the other tests in this file, it seems like we should be sending an error message back (and in general a good way to do it). I think it would be better to keep it like this, as its a better implementation of the controller and the referenced bug for this unit test will still be fixed as the status being returned is still the same

Reply:
I actually agree but any change to functionality is something that could have implications. For example we had a customer complain one time because the content of the error message they received had changed on a 404 and their systems validated a hardcoded message rather than status!

If we are going to go forwards with a change to our response I think that should be signed off by someone outside of development.

Reply:
updated to check for the exception and ignores it if it is